src/components/Player.vue

```diff
@@ -150,6 +153,15 @@
             </template>
             <template v-else> Seat occupied</template>
           </li>
+          <li
+            @click="changePronouns"
+            v-if="
+              !session.isSpectator ||
+                (session.isSpectator && player.id === session.playerId)
+            "
+          >
+            <font-awesome-icon icon="transgender" />Change Pronouns
```

Comment:
I'm currently figuring out if this is a good icon choice for this option. We might have to use another one...

Reply:
yep, agreed.  I am happy to take guidance form anyone with more knowledge or even opinion on this.